axelrod/_strategy_utils.py

```diff
@@ -47,11 +48,23 @@ def limited_simulate_play(player_1, player_2, h1):
     update_history(player_1, h1)
     update_history(player_2, h2)
 
+
 def simulate_match(player_1, player_2, strategy, rounds=10):
     """Simulates a number of matches."""
     for match in range(rounds):
         limited_simulate_play(player_1, player_2, strategy)
 
+
+def calculate_scores(p1, p2, game):
```

Comment:
Does it make sense to just pass in the histories here rather than the player objects?